eclass/vala.eclass

```diff
@@ -15,8 +15,7 @@
 # executables, pkgconfig files, etc., which Gentoo does not provide.
 
 case ${EAPI} in
-	6|7) inherit eutils multilib ;;
-	8) ;;
+	7|8) ;;
```

Comment:
This looks like a real change. Inheriting `eutils` in EAPI=7 is a no-op, so that's fine, but what about multilib?

Reply:
correct, it's a change, the question is, do you really want to audit every consumer or just fix bugs along the way?